src/app/index.js

```diff
@@ -1,117 +1,107 @@
-const gui = require('nw.gui');
```

Comment:
This is some pre-2016 code. You don't need it, `nw` is now globally accessible from `window.nw` and `global.nw`